go/test/endtoend/vtgate/gen4/main_test.go

```diff
@@ -100,6 +104,15 @@ func TestMain(m *testing.M) {
 			return 1
 		}
 
+		uKs2 := &cluster.Keyspace{
+			Name:      unsharded2Ks,
+			SchemaSQL: unsharded2SchemaSQL,
+		}
+		err = clusterInstance.StartUnshardedKeyspace(*uKs2, 0, false)
+		if err != nil {
+			return 1
+		}
+
```

Comment:
we should add comment here to understand why this keyspace was added. Otherwise, without adding a new test it remains unclear.